orderly_web/config.py

```diff
@@ -212,6 +212,13 @@ def __init__(self, path, dat):
             else:
                 raise Exception("web_url must be provided")
 
+        if "ssh" in dat["orderly"] and dat["orderly"]["ssh"]:
+            public = config_string(dat, ["orderly", "ssh", "public"])
+            private = config_string(dat, ["orderly", "ssh", "private"])
+            self.orderly_ssh = {"public": public, "private": private}
```

Comment:
I think you can use `config_dict` here

Reply:
I thought about that, but then we have to validate that the contents of each dict member, and the keys!

Reply:
You mean you don't get the validation that the values are strings? If we're worried about that we shouldn't be using it elsewhere either right (the vault args and the github oauth key/secret should also be strings.)

Reply:
Plus I guess the vault arg keys are variable depending on the auth method, but the github ones are fixed.